src/hooks/useCart/useCart.test.tsx

```diff
@@ -1078,15 +1072,15 @@ describe("useCart", () => {
       });
 
       mockPostTransaction.mockRejectedValueOnce(
-        new Error("error when checking out")
+        new Error("error when checking out.")
```

Comment:
Hmm, should this error be title case?

Reply:
ahh for this case, error message doesnt matter, removed it! thanks!